README.md

```diff
@@ -137,16 +137,18 @@ type Profile = {
   * `scoping`: An optional configuration which implements the functionality [explained in the SAML spec paragraph "3.4.1.2 Element <Scoping>"](https://docs.oasis-open.org/security/saml/v2.0/saml-core-2.0-os.pdf). The config object is structured as following:
 ```javascript
 {
-  idpList: { // optional
-    entries: [ // required
-      {
-        providerId: 'yourProviderId', // required for each entry
-        name: 'yourName', // optional
-        loc: 'yourLoc', // optional
-      }
+  idpList: [ // required
```

Comment:
Is `idpList` really required now? If so, I think that may have been a breaking change that we didn't properly account for.

Reply:
@cjbarth My mistake.

The idpList isn't required as stated in the saml2.0 spec. It actually wasn't required in the previously added feature either as can be seen in `/src/passport-saml/saml/ts:361`.
The only mismatch was in the Typescript interface `SamlScopingConfig`. I reverted the required to optional in the documentation and fixed the Typescript interface in my latest commit.